doc/man7/provider-base.pod

```diff
@@ -405,6 +406,25 @@ The number of bits of security offered by keys in this group. The number of bits
 should be comparable with the ones given in table 2 and 3 of the NIST SP800-57
```

Comment:
Text in the "tls-group-alg" section says "Keys created should be able to support key exchange". Should this say "Keys created should be able to support key exchange or key encapsulation"?

Reply:
Fixed in upcoming commit.